### DISCUSSION TITLE
Hai Reveal [B12] Devil-Hide Battle take off message

### DISCUSSION THREAD

**Discussion by daggs1** [endless-sky/endless-sky#8458] [Q&A]
Greetings,

in the Hai Reveal [B12] Devil-Hide Battle mission, when you take off, you get a hail from Danforth, see https://github.com/endless-sky/endless-sky/blob/master/data/hai/hai%20reveal%205%20philosophers.txt#L1150
is there really any Syndicate reinforcements inbound? I've waited 10 minutes an nothing came

**Comment by mOctave** [comment]
Looking at the mission file:
1. No, you do not get Syndicate reinforcements.
2. However, the fighters and drones that the Republic has are Syndicate-equipped. That appears to be what Danforth means: he says "we just got" not "we're getting," which implies that you just need to wait for all the fighters and drones to arrive at their carriers properly before jumping.
